README

```diff
@@ -185,7 +185,7 @@ options:
   -x                 process xattrs if you are creating gufi dbs of suspect directories
 
 Flow:
-if using input suspect file read that in and put into memory trie for lookups
+if using input suspect file read that in and put into memory tree for lookups
```

Comment:
I think [`trie`](https://en.wikipedia.org/wiki/Trie) is correct here.

https://github.com/mar-file-system/GUFI/blob/1c99a8218d18350b792290bc92b16fbfc7c263cf/utils.h#L91